docs/src/main/sphinx/release/release-479.md

```diff
@@ -0,0 +1,194 @@
+# Release 479 (TBD 2025)
+
+## General
+
+* Support automatic TLS certificate generation in the `ANNOUNCE` node discovery mode. ({issue}`27030`)
+* Add support for specifying the location of the Exchange Manager configuration file. ({issue}`26611`)
+* Publish query finishing time in event listener. ({issue}`27202`)
```

Comment:
Query "finishing time" is an internal concept that has never been exposed to users. It needs to be described in terms of what the user sees.

Reply:
This is a field in the event listener, there are many fields exposed there which have never been explained to end users. FINISHING query state is already visible in the web UI when a query reaches that state, so it's not a complete unknown.
We could say `Publish time taken by query in FINISHING state in event listener`, not sure if that's any better.

Reply:
updated